### ISSUE TITLE
Star (*) search not limited to exact word

### ISSUE THREAD

**Issue by prinz-z** [vrapper/vrapper#750]
When I press * on a word, it does not limit the search to that exact word, but finds words containing the word and get stuck after that.

Example:
We have the following code with cursor on (_) position:

```java
Logger lo(_)g = ...;  // line 1
(...)
// start logging here // line 2
log.error(...);       // line 3
```
When I press * on position (_), it does not jump to line 3 with `log` in it, but goes to line 2 and when pressing * again, it does nothing, because it searches for `logging` now, which is the only occurrence.

**Comment by albertdev** [comment]
Well, I can reproduce it in 0.68.0. I guess this is caused by some recent fix to the regex parser so maybe @keforbes can clarify some more.

**Comment by albertdev** [comment]
Can you share your `.vrapperrc`? I have a feeling this is related to some setting because I cannot reproduce your problem with the latest version.

**Comment by prinz-z** [comment]
I just removed my _vrapperrc for testing purpose, restarted eclipse and I can still reproduce the problem.
But just in case, that's what mainly in the file, while (...) means more definitions similar to the definition above:

```
subwordmappings ,
vmap p "_dP
map gh <C-a>
AddCycleGroup ...
(...)
let @r = ...
(...)
```

**Comment by keforbes** [comment]
@prinz-z, we had a regression in version 0.68.0 which was filed in #747 and fixed in unstable build 0.69.20160610.  Can you upgrade to the unstable build (add unstable update site, check for updates) and let me know if that fixes your problem?

**Comment by prinz-z** [comment]
@keforbes, I updated the unstable plugins and you're right, the problem seem to be fixed. Thank you
